Cookbook/README.md

```diff
@@ -0,0 +1,38 @@
+iOS Cookbook 👩‍🍳
+ ====================================
+
+## I am here because I want to ...
+
+##### ... learn about our architecture 🏠
+
+* [Implementing features with ReactiveFeedback](http://ilya.puchka.me/implementing-features-with-reactivefeedback/)
+* [Global `Current` struct to hold global dependencies](./Proposals/ControlTheWorld.md)
```

Comment:
```suggestion
* [Adopt a `Current` struct to hold global dependencies](./Proposals/ControlTheWorld.md)
```

Reply:
I can use `Adopt`, but it should be `Adopt the `Current` struct to hold`

Reply:
I updated the suggestion to use your suggestion.

![2xqnne](https://user-images.githubusercontent.com/773004/55558956-0f8e6300-56e5-11e9-9c95-4cfdeb9ca4de.jpg)